### ISSUE TITLE
Unable to load TrayIcon plugin

### ISSUE THREAD

**Issue by macias** [lwindolf/liferea#1025]
From all available by default plugin I am unable to load only this one -- TrayIcon. The error states:

> (liferea:12476): libpeas-WARNING **: 18:40:15.429: Error loading plugin 'trayicon'

I tried it on openSUSE 15.3 using KDE3 over generic desktop installation, with Liferea 1.12.9 and also openSUSE Tumbleweed using generic desktop installation with Liferea 1.13.6 -- same result in both cases.

**Comment by lwindolf** [comment]
Was this issue independent of the one described in #1009 ?

**Comment by macias** [comment]
I assume they are independent because the other bug was about showing plugin window, when you select which plugin you would like to load (or not). So before entire plugin window could not be shown, and with the fix you described (once again, thank you!) now it works -- i.e. I can show entire plugin window.

This case is about single, particular plugin. I can load/unload any plugin, except the TrayIcon one.

So in short, this is the same "area" (plugins), but the problem is TrayIcon specific.

**Comment by lwindolf** [comment]
That's hard to debug. If there is no error message above the

     (liferea:12476): libpeas-WARNING **: 18:40:15.429: Error loading plugin 'trayicon'

output I have no idea what it could be.

**Comment by macias** [comment]
I understand. Is there something I could do to trigger more output or something like this?

**Comment by sunwire** [comment]
@macias Is python3-cairo package installed?

**Comment by macias** [comment]
Good call, many thanks, now it works.
